### ISSUE TITLE
Channel topic does not work (for ZK server)

### ISSUE THREAD

**Issue by Licho1** [Spring-Chobby/Chobby#323]
Use !topic to change channel topic and test.

Maybe we don't need topics, so this is just for consideration.

**Comment by GoogleFrog** [comment]
I don't think we need topics. News can be in a dedicated news area or on the site, if required.

**Comment by Licho1** [comment]
We can send site news as a special server message - in welcome for example.
